utils/automatic-release/test/diffToSemver.test.js

```diff
@@ -0,0 +1,37 @@
+const diffToSemver = require('../diffToSemver');
+
+test('return undefined when no change', () => {
+  expect(
+    diffToSemver(
+      { color: '#000000', bg: '#000000' },
+      { bg: '#000000', color: '#000000' }
+    )
+  ).toBeUndefined();
+});
+
+test("return 'major' when there is a breaking change on current keys", () => {
+  expect(
+    diffToSemver(
+      { color: '#000000', bg: '#000000' },
+      { supercolor: '#000000', bg: '#000000' }
+    )
+  ).toBe('major');
+});
```

Comment:
Can we add another test with major being selected in case of
```js
{ color: '#000000', bg: '#000000' },
{ bg: '#000000' }
```
as the first test has addition and removal, and I think it would be nice to also test just the removal?

Reply:
Sure think, good idea